### ISSUE TITLE
Get logging functionality working

### ISSUE THREAD

**Issue by mdbielik** [Team5173/2019-swerve-test#3]
Need to re-enable some logging facility. Maybe see the following for some hints:

https://www.chiefdelphi.com/t/help-w-log4j-on-roborio/148123/2

**Comment by jamiha** [comment]
So the Logging is the angle that is sent to the dashboard correctly. If
that is the case then what am I missing with a line like SmartDashboard.
putNumber("Gyro angle", gyro.getAngle()); if this is wrong should I then
specify a specific angle versus just angle in general like SmartDashboard.
putNumber("Gyro X", gyro.getAngleX());
Thanks for your help whenever you get a chance, Mike.

**Comment by mdbielik** [comment]
No, I'm referring to all of the Logging code for the swerve drive that was
commented out to get it to compile.

We will need some logging capabilities to help us debug our robot code,
especially the swerve drive.

**Comment by jamiha** [comment]
Oh ok, I have the software downloaded with the jar files and stuff I just
need to find how to import the file that has all that into the gradle.
Thanks for the help ill keep searching for how to import slf4j into visual
studio.
